Jenkinsfile

```diff
@@ -3,12 +3,12 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 DOCKER_IMAGE = 'rocm/migraphx-ci-jenkins-ubuntu'
 
 def getgputargets() {
-    targets="gfx906;gfx908;gfx90a;gfx1030;gfx1100;gfx1101"
+    targets="gfx906;gfx908;gfx90a;gfx1030;gfx1100;gfx1101;gfx1201"
     return targets
 }
 
 def getnavi3xtargets() {
-    targets="gfx1100;gfx1101"
+    targets="gfx1100;gfx1101;gfx1201"
```

Comment:
gfx1201 is a navi4 target.  I suspect this "getnavi3targets" intention is to get just Navi3.